extras/rya.prospector/src/main/java/org/apache/rya/prospector/service/ProspectorServiceEvalStatsDAO.java

```diff
@@ -50,10 +54,26 @@ public ProspectorServiceEvalStatsDAO(ProspectorService prospectorService, RdfClo
         this.prospectorService = prospectorService;
     }
 
-    public ProspectorServiceEvalStatsDAO(Connector connector, RdfCloudTripleStoreConfiguration conf) throws AccumuloException, AccumuloSecurityException, TableExistsException {
+    public ProspectorServiceEvalStatsDAO(Connector connector, RdfCloudTripleStoreConfiguration conf) throws AccumuloException, AccumuloSecurityException {
         this.prospectorService = new ProspectorService(connector, getProspectTableName(conf));
     }
 
+    /**
+     * Creates an instance of {@link ProspectorServiceEvalStatsDAO} that is connected to the Accumulo and Rya instance
+     * that is specified within the provided {@link Configuration}.
+     *
+     * @param config - Configures which instance of Accumulo Rya this DAO will be backed by. (not null)
+     * @return A new instance of {@link ProspectorServiceEvalStatsDAO}.
+     * @throws AccumuloException The connector couldn't be created because of an Accumulo problem.
+     * @throws AccumuloSecurityException The connector couldn't be created because of an Accumulo security violation.
+     */
+    public static ProspectorServiceEvalStatsDAO make(Configuration config) throws AccumuloException, AccumuloSecurityException {
```

Comment:
should this be more of a lazy load style dao or is it a new instance every time incase the config changes?